R/pw_info.R

```diff
@@ -185,8 +179,8 @@ pw_info <- function(
     event[, `:=`(
       time = td,
       ln_hr = log(hr),
-      info0 = event * q_c * q_e
-    )]
+      info0 = event * q_c * q_e)]
```

Comment:
Do you prefer the final parenthesis/bracket on the final line? I noticed you changed this in multiple places.

I typically see one of the following two options:

```R
x <- func(arg1 = "a",
          arg2 = "b",
          arg3 = "c")

x <- func(
  arg1 = "a",
  arg2 = "b",
  arg3 = "c"
)
```

Whereas you are using the following:

```R
x <- func(
  arg1 = "a",
  arg2 = "b",
  arg3 = "c")
```

Reply:
Summary of our conversation today:

* @LittleBeannie likes the following syntax because a comment followed by a lot of whitespace breaks up the code into visually-discernable chunks

```R
# Comment
x <- longer_func_name(arg1 = "a",
                      arg2 = "b",
                      arg3 = "c")
```

* However, {styler} automatically enforces the newline after the opening parentheses and before the closing parentheses whenever the arguments can't fit onto a single line. Since {styler} is the most efficient way to apply a consistent coding-style throughout the package, we are going to favor this style

```R
# Comment
x <- longer_func_name(
  arg1 = "a",
  arg2 = "b",
  arg3 = "c"
)
```

* Note that {styler} doesn't enforce one argument per line though. While it is generally best to put one argument per line so that changes are easier to track via Git, it is fine to put multiple args on the same line if they are all short and related, eg

```R
# Comment
x <- longer_func_name(
  x1= "a", x2 = "b", x3 = "c",
  y1 = "d", y2 = "e", y3 = "f",
  z1 = "g", z2 = "h", z3 = "z3"
)
```